### PR TITLE
Sponsors

### PR DESCRIPTION
**Pull request by ayush4345**
### PR description

> Cover things that have been added and/or removed

### CODE REVIEW

**Comment by mittal-parth** [comment]
@anshumanNitk Pls fill the MR description properly.

**Comment by mittal-parth** [comment]
- [x] Resolve conflicts 
- [x] The font style is for some reason wrong on your PR. Check if you have changed it by mistake
- [x] The image size is too big. Reduce it. 
- [x] Remove the white background for all logos that have a white background. Use remove.bg
- [x] Some logos arent good as given by them. For eg, Dairy Day, just pick a better logo from the internet. You have full freedom to change the logo as long as it is the right company xD
- [x] Milkshake Partner,  - too big, name not there
- [x] The best approach I feel, could be to make all images square. That way the weird shape and sizes wont occur
- [x] Add a more button like competitions, shouldn't be scrolling so much
- [x] Add a link to the sponsors on the navbar. 
- [x] Navbar text has also become weird.

**Comment by mittal-parth** [comment]
Go through the checklist once again. Lot of items pending.

File: src/components/Sponsors.jsx [hunk 1]
```diff
@@ -1,8 +1,92 @@
...
+    { src: twin, title: "Women's Lifestyle Partner", name: "Twin Birds" },
+    { src: ims, title: "MBA Coaching Partner", name: "IMS" },
+    { src: Vi, title: "Connectivity Partner", name: "VI" },
+    { src: gatsby, title: "Styling Partner", name: "Gatsby" },
+    { src: golive, title: "E-Sports & Gaming Partner", name: "GoLive Games Studios Pvt. Ltd." },
+
+  ]
+
   return (
-    <div>Sponsors</div>
```
Comment (mittal-parth) [inline src/components/Sponsors.jsx:5]: Give this the same styling as other headings

**Review by mittal-parth — CHANGES_REQUESTED** [review]
Awesome!
- [x] I know not your mistake, but remove Events and I-Care from the navbar.
- [x] Can the sponsor cards have more margin from the left and right on phone. They seem too close to the phone screen end

File: src/components/Sponsors.jsx (hunk 1 above)
Comment (ayush4345) [reply src/components/Sponsors.jsx:5]: that line is removed line.

check line no. 60, it already style like rest of the heading.

**Review by mittal-parth — APPROVED** [review]

**Pull request merged**